src/components/SlateEditor/plugins/blockPicker/SlateBlockPicker.tsx

```diff
@@ -336,33 +387,45 @@ const SlateBlockPicker = ({
         onVisualElementClose={onVisualElementClose}
         onInsertBlock={onInsertBlock}
       />
-      {!visualElementPickerOpen && (
+      <Popover open={blockPickerOpen} onOpenChange={onOpenChange}>
         <Portal>
-          <StyledBlockPickerWrapper ref={portalRef} data-testid="slate-block-picker-button">
-            <SlateBlockMenu
+          <PopoverTrigger asChild ref={portalRef}>
+            <BlockPickerButton
+              colorTheme="light"
               data-testid="slate-block-picker"
-              isOpen={blockPickerOpen}
-              heading={t('editorBlockpicker.heading')}
-              actions={getActionsForArea()
-                .filter(
-                  (action) =>
-                    !action.requiredScope || userPermissions?.includes(action.requiredScope),
-                )
-                .map((action) => ({
-                  ...action,
-                  label: t(`editorBlockpicker.actions.${action.data.object}`),
-                }))}
-              onToggleOpen={(open) => {
-                ReactEditor.focus(editor);
-                setBlockPickerOpen(open);
-              }}
-              clickItem={(data: ActionData) => {
-                onElementAdd(data);
-              }}
-            />
-          </StyledBlockPickerWrapper>
+              aria-label={blockPickerOpen ? t('slateBlockMenu.close') : t('slateBlockMenu.open')}
```

Comment:
Kan vurdere å legge på en `aria-expanded` også her!

Reply:
Det gjøres allerede! Radix gjør det for oss.